### ISSUE TITLE
Dear sir, in your paper, which part of the code is reflected in the input of individual data features from each batch of data into the generator?

### ISSUE THREAD

**Issue by littleJu1y** [xuedongyun/FGLA#1]
I noticed in your code that:def gen_attack_algorithm(grad, y, generator, offset: bool, device):
    conv_out = flg(grad, y, offset)
    conv_out, decoder = conv_out.to(device), generator.to(device)
    imgs = generator(conv_out).detach().cpu()
    return imgs
May I ask con_ out is a feature map of a single data?

**Comment by littleJu1y** [comment]
Addendum to the question: May I ask which specific line of code separated a single data feature map?  Thank you！
